.github/workflows/projects_check.yml

```diff
@@ -55,8 +54,15 @@ jobs:
         show-progress: false
     - name: Install dependencies
       run: python3 -m pip install --user python-debian
-    - name: Validate copyright file
+    - name: Validate copyright file with python-debian
       run: python3 ./utils/check_copyright.py
+    - name: Validate copyright file with debian-control-linter
+      uses: tibetiroka/debian-control-linter-action@v1
```

Comment:
Is your action not doing a superset of the check above?

Reply:
I think so, but since these checks are really fast I don't see an issue with having them both.

Reply:
I guess it would be one less script to maintain. @warp-core thoughts?

Reply:
I think it's fine to keep it for now.